### DISCUSSION TITLE
Is this Addon still being updated?

### DISCUSSION THREAD

**Discussion by VirmosX** [AdiAddons/AdiBags#1023] [Q&A] [answered]
Hey there, I noticed that the Addon still receives updates from @Talyrius. I am very confused now is this addon is still mantained or if it's just random users maintaining it. I am unable to find information regarding this. I can only see the announcement that Adibags will be discontinued. Still Adibags for me works a lot better for me than Betterbags. I am also still missing features i quite enjoyed in Betterbags that Adibags had. Like the button to to sort or disable the 'new' category for items you've just picked up. Can anyone shed a bit of light about the current situation?

**Comment by Cidan** [comment, marked as answer]
No, this addon is not being maintained anymore. If people send PR's, they may get merged in time, but it's not actively developed.

> I am also still missing features i quite enjoyed in Betterbags that Adibags had. Like the button to to sort or disable the 'new' category for items you've just picked up. 

Both of these features have been implemented from almost the start. Are you keeping up with the updates to BetterBags and reading the README? There are only 1-2 major features missing at this point: profiles, and an interactive theme builder.

**Reply by VirmosX** [reply]
Ty for clearing things up. I've not been actively reading the README, but I will now, to get to know all features. Glad that what I am missing is already implemented. Looking forward to the 2 missing features and possible new ones as well. Keep it up!
